### DISCUSSION TITLE
Consulta sobre qué ejercicios son grupales

### DISCUSSION THREAD

**Discussion by Schmidt-mat** [dds-jv/foro#363] [Consultas generales] [answered]
Buenas, quería preguntar qué ejercicios son grupales para la clase que viene, entiendo que como en la entrega tanto del Code smells como el del QMP3 está la opción de poner el grupo son grupales, pero por si acaso pregunto. Además pregunto para confirmar: Todos los ejercios que quedan de tarea MENOS los que son tpi como el de Macowins son grupales verdad?

**Comment by gastonprieto** [comment, marked as answer]
Buenas @Schmidt-mat 

Si mi memoria no falla sos del jueves :P así que si, todos excepto los TPI o alguno que aclaremos que sea individual
